src/app/dropdown/dropdown-angular-close-item-false.demo.html

```diff
@@ -10,14 +10,14 @@
             <clr-icon shape="error" class="is-error" size="24"></clr-icon>
             <clr-icon shape="caret down"></clr-icon>
         </button>
-        <div class="dropdown-menu">
+        <clr-dropdown-menu>
```

Comment:
What we tend to do for this is have the _component_ itself add the CSS class to the host. Less verbosity for the user.